### DISCUSSION TITLE
Suggestion: Tatami mats model better with 5 px hight?

### DISCUSSION THREAD

**Discussion by SonandesMC** [vectorwing/FarmersDelight#747] [Ideas]
When using Farmer's Delight tatami mats, it did not give me the original feeling of stepping up (agaru). In Vanilla I am used to use birch slabs instead: a bed surrounded by birch slabs looks like a futon. So when tatami mats would have a heigt of 5 or 6 pixels instead of 2, the wooden part of a bed would be covered and therefore a bed surrounded by 5px tatami mats looks like a futon as well, as (create-/quark-) seats would look like floor cushions. 
In the screenshot I changed the model in my personal ressource pack accordingly (but not the hitbox). (Damn, I stepped on the tatami in my shoes...)
![2022-05-27_09 12 13](https://user-images.githubusercontent.com/80624262/170650057-bb42af59-4e05-4f8f-964a-7d76bcef4853.png)

**Comment by vectorwing** [comment]
Heyo! Sorry for the very late reply. I moved this over to Discussions.

Hmm... 5 pixels seems a bit much, but I can see what you mean: making them a bit taller could help "transform" certain furniture blocks. Though this is an interesting idea, it would need to be considered carefully, as it would affect all existing builds which use Tatami Mats; they might not be designed for such a high floor profile.

This could be tested for a 0.X version, perhaps.

**Reply by Aceplante** [reply]
I dont really like this proposed change. They're mats, not slabs. I think they're fine as is.
